kusd/api.go

```diff
@@ -159,17 +99,56 @@ func (api *PrivateValidatorAPI) SetCoinbase(coinbase common.Address) bool {
 	return true
 }
 
-// SetDeposit  sets the deposit of the validator
+// SetDeposit sets the deposit of the validator
 func (api *PrivateValidatorAPI) SetDeposit(deposit uint64) bool {
 	api.kusd.SetDeposit(deposit)
 	return true
 }
 
 // GetMinimumDeposit gets the minimum deposit required to take a slot as a validator
-func (api *PrivateValidatorAPI) GetMinimumDeposit() uint64 {
+func (api *PrivateValidatorAPI) GetMinimumDeposit() (uint64, error) {
 	return api.kusd.GetMinimumDeposit()
 }
 
+// GetDepositsResult is the result of a validator_getDeposits API call.
+type GetDepositsResult struct {
+	Deposits []depositEntry `json:"deposits"`
+}
+
+type depositEntry struct {
```

Comment:
`GetDepositsResult` is an exposed struct with an array of `depositEntry`. Shouldn't this also be exposed for consistency?

Reply:
followed the convention in the same file: https://github.com/kowala-tech/kUSD/blob/0c3a346694015e3bea93ec408f2e8ee1365b3ed6/kusd/api.go#L540